core/src/main/java/io/seata/core/rpc/ChannelManager.java

```diff
@@ -197,9 +197,27 @@ private static void updateChannelsResource(String resourceId, String clientIp, S
         if (StringUtils.isNullOrEmpty(dbkey)) {
             return null;
         }
+
+        dbkey = replaceMultipleDbUrl(dbkey, "dbkey_1 = {}", "dbkey_2 = {}");
+
         return new HashSet<String>(Arrays.asList(dbkey.split(Constants.DBKEYS_SPLIT_CHAR)));
     }
 
+    /**
+     * Repair: There is a BUG in delimiting strings when working with multiple database connection addresses
```

Comment:
only need to introduce the function of the method

Reply:
Has been updated

Reply:
> only need to introduce the function of the method
Has been updated